interface/patient_file/summary/demographics_full.php

```diff
@@ -1,1058 +1,1055 @@
-  f.i<?php echo $i?>subscriber_postal_code.value=f.form_postal_code.value;
-  if (f.form_country_code)
-    f.form_i<?php echo $i?>subscriber_country.value=f.form_country_code.value;
-  f.i<?php echo $i?>subscriber_phone.value=f.form_phone_home.value;
-  f.i<?php echo $i?>subscriber_DOB.value=f.form_DOB.value;
-  if(typeof f.form_ss!="undefined")
-    {
-        f.i<?php echo $i?>subscriber_ss.value=f.form_ss.value;  
-    }
-  f.form_i<?php echo $i?>subscriber_sex.value = f.form_sex.value;
-  f.i<?php echo $i?>subscriber_employer.value=f.form_em_name.value;
-  f.i<?php echo $i?>subscriber_employer_street.value=f.form_em_street.value;
-  f.i<?php echo $i?>subscriber_employer_city.value=f.form_em_city.value;
-  f.form_i<?php echo $i?>subscriber_employer_state.value=f.form_em_state.value;
-  f.i<?php echo $i?>subscriber_employer_postal_code.value=f.form_em_postal_code.value;
-  if (f.form_em_country)
-    f.form_i<?php echo $i?>subscriber_employer_country.value=f.form_em_country.value;
- }
-}
-
-<?php } ?>
-
-function popUp(URL) {
- day = new Date();
- id = day.getTime();
- top.restoreSession();
- eval("page" + id + " = window.open(URL, '" + id + "', 'toolbar=0,scrollbars=1,location=0,statusbar=0,menubar=0,resizable=1,width=400,height=300,left = 440,top = 362');");
-}
-
-function checkNum () {
- var re= new RegExp();
- re = /^\d*\.?\d*$/;
- str=document.demographics_form.monthly_income.value;
- if(re.exec(str))
- {
- }else{
-  alert("<?php xl('Please enter a monetary amount using only numbers and a decimal point.','e'); ?>");
- }
-}
-
-// Indicates which insurance slot is being updated.
-var insurance_index = 0;
-
-// The OnClick handler for searching/adding the insurance company.
-function ins_search(ins) {
-    insurance_index = ins;
-    return false;
-}
+  <head>
+    <?php html_header_show();?>
+    <link rel="stylesheet" href="<?php echo $css_header; ?>" type="text/css">
+    <link rel="stylesheet" href="../../../library/css/jquery.datetimepicker.css">
+    <script type="text/javascript" src="../../../library/dialog.js"></script>
+    <script type="text/javascript" src="../../../library/textformat.js"></script>
+    <script type="text/javascript" src="../../../library/js/jquery.datetimepicker.full.min.js"></script>
+    <script type="text/javascript" src="../../../library/js/common.js"></script>
+    
+    <?php 
 
-// The ins_search.php window calls this to set the selected insurance.
-function set_insurance(ins_id, ins_name) {
- var thesel = document.forms[0]['i' + insurance_index + 'provider'];
- var theopts = thesel.options; // the array of Option objects
- var i = 0;
- for (; i < theopts.length; ++i) {
-  if (theopts[i].value == ins_id) {
-   theopts[i].selected = true;
-   return;
-  }
- }
- // no matching option was found so create one, append it to the
- // end of the list, and select it.
- theopts[i] = new Option(ins_name, ins_id, false, true);
-}
+      include_once("{$GLOBALS['srcdir']}/options.js.php"); 
```

Comment:
is options.js.php a native file vs. third-party?  Probably is, just asking.

Reply:
What is the relationship to options.inc.php?

Reply:
I don't really know.

That inclusion wasn't made by me, by the way. It was there already